pygsheets/cell.py

```diff
@@ -171,6 +173,21 @@ def set_text_format(self, attribute, value):
         self.update()
         return self
 
+    def set_text_rotation(self, attribute="angle", value=0):
+        """
+        set the text rotation
+        :param attribute: "angle"
+        :param value: corresponding value for the attribute
+        :return: :class: Cell
+        """
+        if attribute != "angle":
+            raise InvalidArgumentValue("not a valid argument, please see the docs")
+        if type(value) != int:
```

Comment:
Might be better to check the value range (-90, 90), as most like me assumes its (0, 180)